javascript/sbp/acquisition.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (C) 2015 Swift Navigation Inc.
+ * Contact: Joshua Gross <josh@swift-nav.com>
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+// Automatically generated from piksi/yaml/swiftnav/sbp/acquisition.yaml with generate.py.
+// Please do not hand edit!
+
+/**
+ * Satellite acquisition messages from the device.
+ */
+
+var Parser = require('binary-parser').Parser;
+
+var MSG_ACQ_RESULT = function (sbp, msg) {
+  
+  /*SBP class for message MSG_ACQ_RESULT (0x0014).
+  
+
+  
+  This message describes the results from an attempted GPS signal
+acquisition search for a satellite PRN over a code phase/carrier
+frequency range. It contains the parameters of the point in the
+acquisition search space with the best signal-to-noise (SNR)
+ratio.
+
+
+  Parameters
```

Comment:
Also, fixing some of the spacing and alignment will definitely make the source a bit easier to read.